crawl-ref/source/ability-type.h

```diff
@@ -82,7 +82,9 @@ enum ability_type
     ABIL_ZIN_VITALISATION,
     ABIL_ZIN_IMPRISON,
     ABIL_ZIN_SANCTUARY,
+#if TAG_MAJOR_VERSION == 34
     ABIL_ZIN_CURE_ALL_MUTATIONS,
```

Comment:
I assumed removing this would have saved compare issues. Action counts in morgue for example.